pkg/auth/serviceaccount.go

```diff
@@ -66,6 +68,9 @@ func GetOIDCServiceAccountForResource(gvk schema.GroupVersionKind, objectMeta me
 			Annotations: map[string]string{
 				"description": fmt.Sprintf("Service Account for OIDC Authentication for %s %q", gvk.GroupKind().Kind, objectMeta.Name),
 			},
+			Labels: map[string]string{
+				sources.OIDCLabelKey: "",
```

Comment:
nit: we probably want to have an actual value for this label, rather than just an empty string.

Reply:
@Cali0707 Thank you for the comments! Would something like "OIDC label" be a good value for this?